xla/pjrt/gpu/se_gpu_pjrt_compiler.cc

```diff
@@ -199,13 +199,17 @@ StreamExecutorGpuCompiler::Compile(CompileOptions options,
 #endif
 }
 
+#if TENSORFLOW_USE_ROCM
```

Comment:
The nested macro seems to not be supported by MSVC. Pushing the inner ifdef outside the other macro seems to work and doesn't change the behavior/functionality of the code here.